ZLSwipeableViewSwift/ZLSwipeableView.swift

```diff
@@ -62,7 +62,14 @@ open class ZLSwipeableView: UIView {
     open var didEnd: DidEndHandler?
     open var didSwipe: DidSwipeHandler?
     open var didCancel: DidCancelHandler?
-    open var didTap: DidTap?
+    open var didTap: DidTap? {
+        didSet {
+            // Update all viewManagers to listen for taps
```

Comment:
add something like `guard didTap != nil else { return }`? So if a consumer just nils out didTap we don't inadvertently add the tap recognizer?